event/listener.php

```diff
@@ -111,6 +112,22 @@ public function do_poll_modification($event)
 	}
 
 	/**
+	 * Modifys the template vars to match the advanced poll settings
+	 *
+	 * @param object $event The event object
+	 * @return void
+	 */
+	public function do_poll_block_modification($event)
+	{
+		if ($event['s_display_results'] && (int) $event['topic_data']['wolfsblvt_poll_show_ordered'])
```

Comment:
Sure that `$event['s_display_results']` is the best idea? I mean if you haven't voted, this function will not run. But if you vote then via AJAX, the votes are not ordered.
So why not use the same idea like mine, check for `isset($topic_data['poll_title'])` there?
Oh, and the int cast is not needed I would say. If you want typecasting, `(bool)` would be the correct here.